components/post_reminder_time_picker_modal/index.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
nit : Should this be called post reminder custom time picker instead to better deliver its meaning, in first try i was confused why we have this component